src/analysis/retail/priest/shadow/modules/Abilities.tsx

```diff
@@ -16,13 +15,12 @@ class Abilities extends CoreAbilities {
           base: 1500,
         },
         castEfficiency: {
-          suggestion: true,
+          suggestion: false,
           recommendedEfficiency: 0.85,
-          maxCasts: (cooldown: number) =>
-            calculateMaxCasts(cooldown, combatant.getBuffUptime(SPELLS.VOIDFORM_BUFF.id)),
+          //maxCasts: calculateMaxCasts(cooldown, combatant.getBuffUptime(SPELLS.VOIDFORM_BUFF.id)),
         },
         damageSpellIds: [SPELLS.VOID_BOLT.id, SPELLS.VOID_BOLT_DISSONANT_ECHOES.id],
-        enabled: combatant.hasTalent(TALENTS.VOID_ERUPTION_TALENT),
+        enabled: false, //combatant.hasTalent(TALENTS.VOID_ERUPTION_TALENT),
```

Comment:
should this be false? it indicates that the spell can't be cast at all to other systems.

EDIT: oh i see, this is added by the VB module. Probably want to entirely remove this entry.

Reply:
I've removed it from abilities.